tasks/run-matlab-tests/v0/main.ts

```diff
@@ -58,7 +62,11 @@ async function runTests(options: IRunTestsOptions) {
             `'SelectByTag','${options.SelectByTag || ""}',` +
             `'CoberturaModelCoverage','${options.CoberturaModelCoverage || ""}',` +
             `'SimulinkTestResults','${options.SimulinkTestResults || ""}',` +
-            `'PDFTestReport','${options.PDFTestReport || ""}');` +
+            `'PDFTestReport','${options.PDFTestReport || ""}',` +
+            `'UseParallel','${options.UseParallel || ""}',` +
+            `'Strict','${options.Strict || ""}',` +
```

Comment:
`UseParallel` and `Strict` are logicals, so they need to be passed as `true` or `false` not char vectors. Probably want to modify [integ-test-promote.pipeline.yml](https://github.com/mathworks/matlab-azure-devops-extension/blob/master/integ-test-promote.pipeline.yml) to add at least one test using these inputs to make sure they work as expected.

Reply:
Updated, thanks!